### PR TITLE
All strings translated

### PR DESCRIPTION
**Pull request by AJenbo**
This is not intended for being merged but just as a reference for going over the rest of the strings with a fine comb.

### CODE REVIEW

File: Source/control.cpp [hunk 1]
```diff
@@ -1426,7 +1426,7 @@ void DrawChr(const CelOutputBuffer &out)
 	} else {
 		maxdam += plr[myplr]._pDamageMod;
 	}
-	sprintf(chrstr, "%i-%i", mindam, maxdam);
+	sprintf(chrstr, _("%i-%i"), mindam, maxdam);
```
Comment (AJenbo) [inline Source/control.cpp:1429]: Not sure about this, but I guess we will get an error report if needed.

File: Source/control.cpp [hunk 2]
```diff
@@ -1437,7 +1437,7 @@ void DrawChr(const CelOutputBuffer &out)
 	else
 		col = COL_BLUE;
 	if (plr[myplr]._pMagResist < MAXRESIST) {
-		sprintf(chrstr, "%i%%", plr[myplr]._pMagResist);
+		sprintf(chrstr, _("%i%%"), plr[myplr]._pMagResist);
```
Comment (AJenbo) [inline Source/control.cpp:1440]: Probably not needed either.

File: Source/utils/language.cpp [hunk 3]
```diff
@@ -64,15 +64,15 @@ bool parse_metadata(char *data)
 	char *ptr = data;
 	bool utf8 = false;
 
-	while (ptr && (delim = strstr(ptr, ":"))) {
+	while (ptr && (delim = strstr(ptr, _(":")))) {
```
Comment (AJenbo) [inline Source/utils/language.cpp:67]: Ha ha, I love the idea of the translation file parse being translated :joy:

File: Source/control.cpp (hunk 1 above)
Comment (altiereslima) [reply Source/control.cpp:1429]: #: Source/control.cpp:1916

Comment (altiereslima) [reply Source/control.cpp:1429]: ADD_PlrStringXY(out, 366, 121, 600, "you want to remove?", COL_GOLD);

Comment (AJenbo) [reply Source/control.cpp:1429]: Thanks, i have added it.

**Pull request closed without merging**